plugins/mcreator-localization/help/fr_FR/villagerprofession/hat.md

```diff
@@ -1 +1,13 @@
-Définit le type de chapeau qu'ont les villageois avec cette profession.
\ No newline at end of file
+Ce paramètre spécifie le type de chapeau défini par la texture de métier sélectionnée, qui contrôle
+si le villageois portera toujours le chapeau défini par son type après avoir revendiqué cette profession :
+* **AUCUN :** Dans tous les cas ;
+* **PARTIEL :** Dans les cas où il ne couvre pas toute la tête du villageois ;
+* **COMPLET :** En aucun cas.
```

Comment:
What needs to be done here:
1. Replace French translations of uppercased words (parameter values) with the original ones, like this:
```markdown
* **None (aucun):** Dans tous les cas;
* **Partial (partiel):** Dans les cas où il ne couvre pas toute la tête du villageois;
* **Full (complet):** En aucun cas.
```
(Also I guess these values should only have first letter uppercased, this one applies to other languages too)
2. Remove whitespaces before punctuation marks (`profession ;` -> `profession;`)
3. Format the table using IDEA (open help tip inside it and it should warn about incorrect formatting)

Reply:
Done!